src/ComboBox.tsx

```diff
@@ -1,6 +1,10 @@
 import type { ComboBoxDefaultItem } from '@vaadin/combo-box';
 import { ComponentType, type ForwardedRef, forwardRef, type ReactElement } from 'react';
-import { ComboBox as _ComboBox, WebComponentModule, type ComboBoxProps as _ComboBoxProps } from './generated/ComboBox.js';
+import {
+  ComboBox as _ComboBox,
+  type WebComponentModule,
```

Comment:
Does this mean `WebComponentModule` becomes a type-only export also? Is this intentional?

Reply:
Oops